config/skim_UL2016_mib.cfg

```diff
@@ -3,15 +3,15 @@
 # do not select events with at least two jets in the final state
 # false : only >= 2 b jet ; true : all events
 ## for sync use beInclusive==true, for full porduction use false
-beInclusive = false
+beInclusive = true
 #use deepFlavor to order jets
 useDeepFlavor = true
 # Store only ETau/MuTau/TauTau events in the skims
-onlyFinalChannels = true
+onlyFinalChannels = false
```

Comment:
maybe keep true (now the skimmer also includes mumu)?

Reply:
I think the point of it was really to remove the MuMu channel which is quite time consuming so I would modify the line https://github.com/LLRCMS/KLUBAnalysis/blob/37a9ee2451806f47c019647dfde7eba29a97b290/test/skimNtuple_HHbtag.cpp#L2152 to deal with pair types 0,1,2 and leave the flag in the config file so that we can remove the MuMu channel when it is not needed.

Reply:
We do need mumu all the time to measure the MET SFs, so I would make the inclusion of mumu the default. And it can also be used for defining CRs.